server/routing/lobby.go

```diff
@@ -121,23 +121,24 @@ func (lr *LobbyRoutes) lobbyStartGameHandler(w http.ResponseWriter, r *http.Requ
 		http.Error(w, fmt.Sprintf("no lobby found with ID '%v'", lobbyID), http.StatusNotFound)
 	}
 
+	gameInstance := lr.Services.Game.CreateGame(lobby)
+
 	msg := pogo.GameStartMsg{
 		GameID:  lobby.ID,
 		Players: lobby.Players,
-		Msg:     "The game is starting.",
+		GameState: gameInstance.State,
+		Msg:     "The game has started.",
```

Comment:
If this is a GameStartMsg, do we need a string on it that says "The game has started."? Seems redundant.